pkg/flow/internal/controller/component_references.go

```diff
@@ -48,7 +48,7 @@ func ComponentReferences(cn dag.Node, g *dag.Graph) ([]Reference, diag.Diagnosti
 		//
 		// Any call to an stdlib function is ignored.
 		var emptyScope vm.Scope
-		if _, ok := emptyScope.Lookup(t[0].Name); ok && len(t) == 1 {
```

Comment:
I'm not sure what was with this check before, I think I was worried that there may eventually be something in the standard library whose name matches a namespace of a Flow component, but I don't think this is practically going to happen. 

I removed this check so that `constants.*` worked properly, since otherwise it thought `constants.hostname` was a reference to a component.